nipype/utils/spm_docs.py

```diff
@@ -59,5 +59,5 @@ def _strip_header(doc):
         except ValueError:
             index = len(doc)
         return doc[:index]
-    except KeyError as e:
+    except ValueError as e:
```

Comment:
I think this whole block might be better rewritten as

```Python
    try:
        index = doc.index(hdr)
    except ValueError as e:
        raise_from(IOError('This docstring was not generated by Nipype!\n'), e)
    index += len(hdr)
    index += 1
    doc = doc[index:]
    try:
        index = doc.index(cruft)
    except ValueError:
        index = len(doc)
    return doc[:index]
```

This helps make clear where the exception is expected, and avoids catching unexpected exceptions.